src/main/java/com/mercateo/sqs/utils/visibility/VisibilityTimeoutExtenderFactory.java

```diff
@@ -33,6 +40,13 @@ public class VisibilityTimeoutExtenderFactory {
 
     private final AmazonSQS sqsClient;
 
+    private static final Retryer<ChangeMessageVisibilityResult> retryer = RetryerBuilder
+            .<ChangeMessageVisibilityResult>newBuilder()
+            .retryIfExceptionOfType(SdkClientException.class)
```

Comment:
A retry doesn't help for many SdkClientException situations e.g. ReceiptHandle expired, ReceiptHandle is invalid ....
You can check the cause of the SdkClientException, see also [review suggestions](https://github.com/Mercateo/sqs-utils/pull/19/commits/230e31b360c31e5a2c8ff3502cfea33ea1805728#diff-1eba36121d426a3d75d856aff4785c9166919a374541962987767d47769b93f7R59)